### ISSUE TITLE
Displayed wielded item is buggy

### ISSUE THREAD

**Issue by Wuzzy2** [minetest-mods/unified_inventory#135]
The displayed wielded item is buggy. MT 5.0.1

1) The displayed text overlaps with the statbars
2) If `show_wielded_mod` is present, wielded item is shown twice

Alternatives:

Remove this feature (#101).

**Comment by Wuzzy2** [comment]
Version: 04b1cb9e7d748b4a2366ce321ac5a9f76e250a57

**Comment by VanessaE** [comment]
The item name is shown only once on my client both in singleplayer and on all my servers that use it, though it does overlap the statbars.

Solution:  move the item name up a bit.

**Comment by Wuzzy2** [comment]
Oops. I meant the `show_wielded_item` mod here: https://forum.minetest.net/viewtopic.php?t=18011

Are you sure this does not happen? If I have both `unified_inventory` and `show_wielded_item` activated, the text totally displays twice for me.

**Comment by SmallJoker** [comment]
Please add a setting or optional dependency to u_i if you really want to disable it.

**Comment by VanessaE** [comment]
I don't have that mod installed, but why would you install it if UI will already show the item name?

**Comment by Wuzzy2** [comment]
Because in UI it's broken? :D

IMHO this feature should never have been included in UI in the first place. It's not part of the inventory, after all.

Suggestion: `if minetest.get_modpath("show_wielded_item") == nil then --[[ do item_name.lua stuff ]] else --[[ don't do it ]] end`

**Comment by VanessaE** [comment]
If a feature is broken, the correct action is to fix it, not replace it with some other feature/mod.

**Comment by SmallJoker** [comment]
Setting added.
